specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/disasterRecoveryConfigs.json

```diff
@@ -425,6 +425,18 @@
               "type": "string",
               "description": "ARM Id of the Primary/Secondary eventhub namespace name, which is part of GEO DR pairing"
             },
+            "type": {
+              "type": "string",
+              "enum": [
+                "MetadataReplication"
+              ],
+              "x-ms-enum": {
+                "name": "type",
+                "modelAsString": false
```

Comment:
modelAsString must be true, please generate the SDK and see the difference between modelAsString false and true.